.github/workflows/e2e-tests.yml

```diff
@@ -202,6 +202,8 @@ jobs:
 
     strategy:
       matrix:
+        php-version: ["8.1"]
```

Comment:
This doesn't do what you think it does. See: https://github.com/phpstan/phpstan-src/actions/runs/8159444153/job/22303754862 - you basically disabled all of the tests.

Normal run: https://github.com/phpstan/phpstan-src/actions/runs/8170798487

Reply:
good catch. fixed it.

thank you